validateIntrospectorScript.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright (c) 2022, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+found_error=0
+
+declare -a filesToCheck=("src/main/resources/scripts/introspectDomain.sh" 
+                "src/main/java/oracle/kubernetes/operator/ProcessingConstants.java"
+                "src/main/resources/scripts/introspectDomain.py"
+                )
+for fileToCheck in "${filesToCheck[@]}"
+do
+  if [[ ! `grep "Domain introspection complete" "${fileToCheck}"` ]]; then
+    printf "Failed to find the expected string 'Domain introspection complete' in '"${fileToCheck}"' file.\n"
+    exit 1
+  fi
+done
+
+found_error=$?
+
+exit $found_error
```

Comment:
minor: i think this can be now just 'exit 0' -- it looks found_error will always be 0, so found_error can be removed...

Reply:
Changed in [f1b32d4](https://github.com/oracle/weblogic-kubernetes-operator/pull/3408/commits/f1b32d424f5616d6361958594227947e979c2fea).